coralproject/talk
# [next] Static URI Support

## What does this PR do?

Adds support for a new `STATIC_URI` configuration option that can be used when you want to serve Talk along with a CDN capable of caching static assets.

## How do I test this PR?

1. Build the static assets via `npm run build`
2. Use a tool like ngrok to create a new url that isn't on localhost.
3. Copy the URL generated for ngrok to your `.env` file as `STATIC_URI=https://<ngrok url`.
4. View the Talk stream, notice that the assets are properly prefixed with the `{{ staticURI }}` template variable.
